### ISSUE TITLE
[Suggeston]  The doors in the sacrifice room must close when the angels are spawned 

### ISSUE THREAD

**Issue by epfly6** [UnderscoreKilburn/repentance-issues#925]

**Comment by rat-rar** [comment]
goobd idea

**Comment by Pezbi** [comment]
PLEASE! I’m so sick of this bug!!!

**Comment by krispypants** [comment]
Fighting uriel and having that fight mandatory before getting the 11th payout is great since everytime you would just leave the room before  uriel even ends its spawn animation so you can get the coins/soul heart payout, it would make it your reward for fighting that boss and it would fix the consistency error with enemy spawning since every time an enemy spawns through a result of something the doors close like pressing a button or challenge rooms, sac rooms are the only place where spawned enemies don't close doors and I think it's weird for it to be like that.

**Comment by pelzz** [comment]
Nah, the angels were always just an optional thing to be able to get the key pieces, and this would cause a lot of unneeded distractions, especially if you're just trying to use the sacrifice room for the rewards and don't care about the key peices, casual players farm sacrifice rooms a lot so this would just be a chore to do for them, especially earlier in the game fighting uriel with not a whole lot of damage, the troll bombs spawned from the previous wave are not always so reliable to be able to do a lot of damage to uriel and they can even hit you, and spawn hosts from skulls (if there were any) if you didn't blow them up beforehand, so you'd be forced to fight uriel with hosts, and not a lot of damage, does not sound like a pleasant thing at all if you ask me, it's pretty much fine with the way it is currently.

**Comment by Turtier** [comment]
Also, its not true that all types of enemies spawning close doors, a prime example being the hosts from blowing up skulls. I do think its annoying to sometimes leave the room on accident while fighting, but that annoyance is a lot more uncommon than the potential annoyance of being forced to fight uriel every time. Maybe have a middle ground, like locking the door after 3 seconds, so you have time to leave immediately and skip the fight, but you won't accidentally leave if you've chosen to fight?
